src/components/tabset/tab/tab.component.ts

```diff
@@ -31,6 +31,9 @@ export class TabComponent implements OnInit, OnDestroy {
     /** Define the tab heading */
     @Input() heading: string;
 
+    /** Define the tab router path */
+    @Input() routerPath: string;
```

Comment:
Having thought about it, I would prefer `route` as the name of this input, as long as it doesn't cause any conflicts.